src/session.ts

```diff
@@ -32,6 +32,9 @@ export interface Session {
 export const createState = (initial: State) => {
```

Comment:
Perhaps this should now be renamed to `loadState`, the param now acts as a default.